components/operator/api/v1alpha1/dockerregistry_types.go

```diff
@@ -21,13 +21,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type Endpoint struct {
```

Comment:
Whether CR documentation will be updated (`docs/user/resources/06-20-docker-registry-cr.md`)?

Reply:
Yes, in the last PR in the chain